### ISSUE TITLE
[ClojureStorm] multimethods defined with alias are not instrumented

### ISSUE THREAD

**Issue by maksut** [jpmonettas/flow-storm-debugger#94]
I'm not sure if this is a bug or a known limitation.

Given a simple multimethod with a definition in a separate namespace, such as:

item/def.clj
```
(ns item.def)

(defmulti testing :type)
```

item/impl.clj
```
(ns item.impl (:require [item.def :as id]))

(defmethod id/testing :bar [{value :value}] (dec value))
```

test.clj
```
(ns test (:require [item.def]
                   [item.impl]))

(comment
  (require '[flow-storm.api :as fs-api])
  (fs-api/local-connect)

  (item.def/testing {:type :bar :value 123}))
```

After evaluating the rich comment contents, the call `item.def/testing` does not appear in call stack tree / code / function tools.

But if I repeat the same thing without `:as` in `imp.clj` then the multimethod appears. Such as:

item/impl.clj
```
(ns item.impl (:require [item.def]))

(defmethod item.def/testing :bar [{value :value}] (dec value))
```

**Comment by jpmonettas** [comment]
Good catch @maksut ! Indeed it was a bug. 

It is fixed now in `1.12.0-alpha4_3` and `1.11.1-9`. 

You can see the fix here : https://github.com/jpmonettas/clojure/commit/2d71b0c6e63b80823a4e45af6a3ab05fb00b4a22

On other notes, when using ClojureStorm you don't need to require anything or call local-connect, you can start the UI whenever you need it by evaluating the `:dbg` keyword, which is more convenient since you can do it in any namespace without needing to require anything.

**Comment by maksut** [comment]
That was a speedy fix! I can confirm that it is fixed in `1.11.1-9`.

This is a really cool tool. Thanks for that too.
